uiowa/uids
# Feature: Card mobile image sizes and borders fixes

Resolves https://github.com/uiowa/uids/issues/448. 

# How to test

Go to https://localhost:3000/components/preview/card--list or https://uids.brand.uiowa.edu/branches/feature_card_image_mobile/components/preview/card--list.html and view on mobile emulator and desktop and verify that:
- Image circle ssizes appear the same on mobile and desktop `small`, `medium`, and `large` for the circle option. 
- Borders are only applied on mobile and desktop if `card--enclosed` is attached the card (previously we were attaching borders for mobile no matter how the card was configured).
